iterators.tex

```diff
@@ -408,7 +409,7 @@
 \begin{itemize}
 \item \tcode{++i} is valid if and only if \tcode{i++} is valid.
 \item If \tcode{i} is incrementable, then both \tcode{++i}
-  and \tcode{i++} advance \tcode{i} to the next element.
+  and \tcode{i++} advance \tcode{i} to denote the next (possibly past-the-end) element.
```

Comment:
> I'm inclined to just back this change out for now and come back to the committee with a proposal when we'v thought more deeply about it.

Agreed. I know I've had this "remove `WeaklyIncrementable`" argument with myself before, and there doesn't seem to be a good choice.